src/hooks/use-inscription.js

```diff
@@ -19,11 +19,12 @@ function useInscription(inscriptionId) {
     setNostrData(data);
   };
 
-  const fetchInscription = async () => {
-    if (!currentInscriptionId) return;
-    console.log("[useInscription]", currentInscriptionId);
+  const fetchInscription = async (id) => {
+    const _currentInscriptionId = id || currentInscriptionId;
```

Comment:
@topether21 the issue is that, for some reason, currentInscriptionId is being undefined, even though setCurrentInscriptionId has been called previously. Since it requires a change of state it doesn't trigger the function when it need it. 
The fix is to bypass that cycle, and use the inscription id directly from the hook argument.